admin/css/views/stats.css

```
@@ -570,19 +605,83 @@
+            color-mix(in srgb, var(--md-sys-color-error-container) 92%, transparent) 46%,
+            color-mix(in srgb, #ffe0b2 42%, transparent) 74%,
+            transparent 100%
+        ),
+        var(--md-sys-color-surface);
+    /* 描边权重对齐通用 .card（1px glass 级），色相跟随暖色 */
+    --card-border-color: color-mix(in srgb, var(--md-sys-color-error) 18%, transparent);
+    --card-hover-border-color: color-mix(in srgb, var(--md-sys-color-error) 32%, transparent);
+    --card-shadow: 0 4px 20px 0 color-mix(in srgb, var(--md-sys-color-error) 10%, transparent);
+    --card-hover-shadow: 0 12px 30px 0 color-mix(in srgb, var(--md-sys-color-error) 16%, transparent);
+    --hover-lift: 4px;
     position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)